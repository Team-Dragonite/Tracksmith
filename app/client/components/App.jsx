import React from 'react';
import { Routes, Route, Link } from 'react-router-dom';
import SignInPage from './SignInPage.jsx';

function App() {
  return (
  // <Routes>
    <SignInPage />
  // </Routes>
  );
}

export default App;
