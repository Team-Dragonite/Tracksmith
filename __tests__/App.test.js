/**
 * @jest-environment jsdom
 */


 import React from 'react';
 import '@testing-library/jest-dom'
 import renderer from 'react-test-renderer';
 import App from '../app/client/components/App';
 import { MemoryRouter } from 'react-router-dom';
 
 it('renders App', async () => {
   const data = { description: "" }
   const component = renderer.create(
     <MemoryRouter>
       <App /></MemoryRouter>,
   );
   let tree = component.toJSON();
   await expect(tree).toMatchSnapshot();
 });
 