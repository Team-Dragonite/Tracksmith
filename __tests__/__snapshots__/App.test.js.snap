// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App 1`] = `
Array [
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-l6hioc-MuiGrid-root"
  >
    <div
      className="MuiBox-root css-0"
    >
      <div
        className="MuiBox-root css-0"
      >
        <div
          className="MuiTypography-root MuiTypography-h1 MuiTypography-gutterBottom css-a4d0r7-MuiTypography-root"
        >
          TrackSmith
        </div>
      </div>
      <div
        className="MuiBox-root css-0"
        style={
          Object {
            "justifyContent": "space-between",
            "textAlign": "center",
          }
        }
      >
        <div
          className="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
        >
          <label
            className="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
            data-shrink={false}
            htmlFor="outlined-basic"
            id="outlined-basic-label"
          >
            Username
          </label>
          <div
            className="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
              disabled={false}
              id="outlined-basic"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="text"
            />
            <fieldset
              aria-hidden={true}
              className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                className="css-173wfxe"
              >
                <span>
                  Username
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
      </div>
      <br />
      <div
        className="MuiBox-root css-0"
        style={
          Object {
            "justifyContent": "space-between",
            "textAlign": "center",
          }
        }
      >
        <div
          className="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
        >
          <label
            className="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
            data-shrink={false}
            htmlFor="outlined-basic"
            id="outlined-basic-label"
          >
            Password
          </label>
          <div
            className="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
              disabled={false}
              id="outlined-basic"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="password"
            />
            <fieldset
              aria-hidden={true}
              className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                className="css-173wfxe"
              >
                <span>
                  Password
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
      </div>
      <br />
      <div
        className="MuiBox-root css-0"
        style={
          Object {
            "justifyContent": "space-between",
            "marginLeft": "20",
            "marginRight": "20",
            "textAlign": "center",
          }
        }
      >
        <button
          className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root  css-sghohy-MuiButtonBase-root-MuiButton-root"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "marginLeft": "20",
              "marginRight": "20",
            }
          }
          tabIndex={0}
          type="button"
        >
          Log In
        </button>
        <a
          href="/signup"
          onClick={[Function]}
        >
          <button
            className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root  css-sghohy-MuiButtonBase-root-MuiButton-root"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "marginLeft": "20",
                "marginRight": "20",
              }
            }
            tabIndex={0}
            type="button"
          >
            Sign Up
          </button>
        </a>
      </div>
    </div>
  </div>,
  "",
]
`;
